apps/website/src/content/docs/code.mdx

```diff
@@ -1,18 +1,17 @@
 ---
 title: Code
-description:
-propsPath: '@itwin/itwinui-react/esm/core/Typography/Code.d.ts'
+description: Showcase code snippets in line with the body text.
 thumbnail: Code
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='code' />
-
 <LiveExample src='Code.main.tsx' truncate={false}>
   <AllExamples.CodeMainExample client:load />
 </LiveExample>
 
+To emphasize a code snippet, use inline code. This makes it more recognizable and easier for the reader to identify.
```

Comment:
Where are you getting this idea from? It's not used for "code snippets". This is a code snippet:
```js
const sum = 2 + 2;
```

Have you researched and thought about what the `<code>` element does before submitting the PR?

Reply:
Yeah that's the one. Similar to the `<kbd>` element, we can explain that it wraps `<code>`, link to MDN, and include a brief description of what it does and when to use it. Nothing complicated.